ddp-workspace/projects/ddp-angio/app.yaml

```diff
@@ -1,6 +1,11 @@
 service: angio
 runtime: python37
 
+# This setting will not matter except to prevent default automatic-scaling which prevents disabling
+# earlier versions of this service
+basic_scaling:
+  max_instances: 1
+
```

Comment:
As comment indicates, this option allows us to use the `gcloud app deploy --stop-previous-version` flag.